tensorboard/plugins/scalar/tf_scalar_dashboard/tf-scalar-chart.html

```diff
@@ -200,7 +201,25 @@
         runs: Array,  // of String
         tag: String,
 
-        /** @type {X_TYPE} */ xType: String,
+        /**
+         * Allows:
```

Comment:
I'm not strongly opposed to this comment, but I really don't see what value it adds. The names "step", "wall_time", "and "relative" are pretty self-explanatory, and they are defined on the `X_TYPE` object above. The code documents itself, IMO.